### DISCUSSION TITLE
I modified the source code of web-apps. How do I compile installer for linux with .deb extension?

### DISCUSSION THREAD

**Discussion by carlosherz192** [ONLYOFFICE/DocumentServer#2732] [Q&A]
Hello I want to modify the source code of webapps for personal use and after I complete the modification I compile the code following the steps:
[Compiling ONLYOFFICE Docs for a local server](https://helpcenter.onlyoffice.com/installation/docs-community-compile.aspx)
With the previous steps I can compile and run the code, however I want to deploy it in docker but I have to generate a linux installation package with a .deb extension, so I would like to know if there is a manual, script or toturial to be able to create the installable package and be able to do it deploy with docker.

Any idea or suggestion would be very helpful.

Thanks.

**Comment by jiriks74** [comment]
> however I want to deploy it in docker

Go to the [Docker-DocumentServer](https://github.com/ONLYOFFICE/Docker-DocumentServer) repository and modify the [`Dockerfile`](https://github.com/ONLYOFFICE/Docker-DocumentServer/blob/40efc5bea9d7c652f6e00a2a95e7027c1130f64b/Dockerfile#L87) to use your local `deb` instead of the one that is downloaded from the release page.

**Reply by carlosherz192** [reply]
Thanks for reply, I understand that I have to modify the [Dockerfile](https://github.com/ONLYOFFICE/Docker-DocumentServer/blob/40efc5bea9d7c652f6e00a2a95e7027c1130f64b/Dockerfile#L87) from [Docker-DocumentServer](https://github.com/ONLYOFFICE/Docker-DocumentServer) repository, the question is How I build deb package from DocumentServer compiled code? I follow the steps [Compiling ONLYOFFICE Docs for a local server](https://helpcenter.onlyoffice.com/installation/docs-community-compile.aspx) but it does not generate the deb package
